_posts/2020/2020-02-06-getting-new-employees-into-scrum.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Getting new employees into Scrum 
+date: 2020-02-06
+header_image: public/scrum-intro-session.jpg
+header_overlay: true
+category: methods-and-tools
+tags: ["scrum", "agile", "remote"]
+authors: ["Aroso"]
+about_authors: ["amahaiuddin"]
+---
+
+ePages is already [working agile for several years](/blog/methods-and-tools/epages-turns-agile/){:target="_blank"}.
+In an agile company, it is important that the employees know what Scrum means, and also how agile is lived at ePages.
+But just explaining Scrum to new employees in two or three sentences has maybe not the biggest learning effect.
+That's why the Scrum Masters at ePages do an extra session introducing new employees to Scrum during their onboarding time.
+Starting as a working student at ePages, I never experienced the Scrum framework in a company before.
+So, for me it was a great opportunity to learn more about Scrum.
+But how does a Scrum session for new employees look like?
+
+## Introduction
+
+The Scrum session was hosted by a Scrum Master of ePages who organized the whole session.
+In total four new employees participated – three employees from our Hamburg office and one employee from our office in Jena.
+Since the session was hold in Hamburg, our employee from Jena participated remotely via Google Hangouts.
+We were a mixed group of Developers and Technical Writers.
+In the beginning of the session all participants including our Scrum Master introduced themselves shortly:
+
+- What's your name?
+- Which job position do you have and what's your background?
+- Which team do you belong to and what are your tasks?
+
+## Learning by playing
+
+Only sitting there for an hour and listen to what Scrum is about would be way too boring.
+So, what our Scrum Master prepared was a little game in the beginning.
+We as a team got the following task: We got a double deck of cards (each deck of cards contains 52 cards) which we had to sort by card colour (red and black), card type (hearts, diamonds, spades and clubs) and order their card value from low (one) to high (ace).
+Our colleague in Jena got half a deck of cards to keep it fair.
+Before we started to sort the cards, we had to discuss in the team how much time we think we will need for the task.
+Then we had two minutes time to discuss in the team how to solve the task:
+
+- Who is doing what?
+- How we do proceed?
+- With which strategy do we solve the task (in) the best possible way?
+
+After we clarified our strategy, we started to sort the cards and our Scrum Master measured the time.
+In the first round neither we solved the task in the estimated time, nor we sorted all cards in the correct order.
+We played three rounds in total.
+After each round we talked about where our mistakes have been, how we can avoid the mistakes and how we can improve our strategy.
```

Comment:
After each round we talked about the mistakes we made and how we can avoid them to improve our strategy.